.github/workflows/publish-abi_wasm.yml

```diff
@@ -42,10 +41,11 @@ jobs:
         run: |
           mkdir temp_publish_dir
           cp -r ${{ env.BUILD_OUTPUT_PATH }}/* temp_publish_dir/
+          touch temp_publish_dir/yarn.lock
```

Comment:
Why is this needed?

Reply:
`yarn npm publish` will throw an error otherwise, as temp_publish_dir is not part of the workspaces.
The accepted solution for this is to create an empty yarn.lock so that yarn understands that the project is separate from the monorepo.

Reply:
An alternative would be to copy those files back to originating folders and publish from there?
Similar to this:
[how integration test builds packages](https://github.com/noir-lang/noir/blob/8640bc77380b0abea888ad8e7c3dc6bb2f0d7c7c/.github/workflows/test-integration.yml#L136)

Reply:
I think that might fail due to the nature of nix's /result folder. In your example we are not generating a .tar inside of nix's result folder.